pkg/setting/setting.go

```diff
@@ -1633,6 +1636,8 @@ func readAccessControlSettings(iniFile *ini.File, cfg *Cfg) {
 	cfg.RBACPermissionValidationEnabled = rbac.Key("permission_validation_enabled").MustBool(false)
 	cfg.RBACResetBasicRoles = rbac.Key("reset_basic_roles").MustBool(false)
 	cfg.RBACSingleOrganization = rbac.Key("single_organization").MustBool(false)
+	// TODO @eleijonmarck: remove me once we have migrated to actionsets
+	cfg.OnlyAccessActionSets = rbac.Key("only_access_action_sets").MustBool(false)
```

Comment:
Nit: not sure about the config option name. I think I would prefer `only_store_action_sets`.

Reply:
`only_store_action_sets` , `only_use_action_sets`, `force_action_sets`

Reply:
changed to `only_store_access_action_sets`